buildmaster/ATLAS_1JET_13TEV_DIF/metadata.yaml

```diff
@@ -0,0 +1,81 @@
+# implemented by Tanishq Sharma
+
+setname: "ATLAS_1JET_13TEV_DIF"
+
+
+nnpdf_metadata:
+  nnpdf31_process: "JETS"
+  experiment: "ATLAS"
+
+arXiv:
+  url: "https://arxiv.org/abs/1711.02692"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1634970"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1634970"
+  version: 1
+
+version: 1
+version_comment: "NA"
+
+implemented_observables:
+
+  - observable: {description: "double differential cross section w.r.t. pT and y", label: '$\frac{d^2\sigma}{dp_T dy}$', units: "pb/GeV"}
+    observable_name: PT-Y
+    ndata: 177 # 33+32+31+31+28+22
+    tables: [1, 2, 3, 4, 5, 6]
+    kinematics:
+      variables:
+        y: {description: "jet absolute rapidity", label: "$y$", units: ""}
+        pT_sqr: {description: "jet transverse momentum squared", label: "$p_T^2$", units: "GeV^2"}
+        sqrt_s: {description: "center of mass energy", label: '$\sqrt{s}$', units: "GeV"}
+      file: kinematics.yaml
+    data_central: data.yaml
+    data_uncertainties:
+      - uncertainties.yaml
+    kinematic_coverage: [y, pT_sqr, sqrt_s]
+    plotting:
+      kinematics_override: identity
+      dataset_label: 'ATLAS Jet 13 TeV: $\frac{d^2\sigma}{dp_T dy}$'
+      plot_x: pT_sqr
+      figure_by: 
+        - y
+    theory:
+      FK_tables:
+        - - ATLAS_1JET_13TEV_DIF_PT-Y_BIN1
+          - ATLAS_1JET_13TEV_DIF_PT-Y_BIN2
+          - ATLAS_1JET_13TEV_DIF_PT-Y_BIN3
+          - ATLAS_1JET_13TEV_DIF_PT-Y_BIN4
+          - ATLAS_1JET_13TEV_DIF_PT-Y_BIN5
+          - ATLAS_1JET_13TEV_DIF_PT-Y_BIN6
+      operation: 'null'
+
+  - observable: {description: "double differential cross section w.r.t. pT and y - altcorr1", label: '$\frac{d^2\sigma}{dp_T dy}$', units: "pb/GeV"}
+    observable_name: PT-Y-ALTCORR1
```

Comment:
This is a question also for @enocera:  PT-Y and PT-Y-ALTCORR1, should the data be different if the difference is only the uncertainties? Shouldn't this be a variant (i.e., shouldn't the data be the same, I see that the numerical values are different...)

For @t7phy, even if they are separate observables (if the data changes then they are I'd say)I  can see the kinematics are exactly the same, you can use just one of them and remove the other.

Btw, you should be able to do that for the uncertainties as well. _if I did everything correctly_ then uncertainties get concatenated so if one of them is shared you can have them in a separate file an add that file in both :)

Reply:
I can answer this, they are indeed the same datasets with different uncertainties and ideally should have variants for the uncertainties. There was a caveat, in that the uncertainties were not symmetrical. When you symmetrize them, the data point shifts a bit, and this resulted in having to treat them separately.

Reply:
ahh, okok

Then perfect. The comment remains about the kinematics/uncertainties. I haven't look at them to check whether they are numerically equal, but if they are you can remove one file (like the `x_scale` this as an aesthetical point of course, the commondata works the same).

Reply:
No, the uncertainties are indeed different, thats why their symmetrization changes the the data point. The kinematics on the other hand is the same, but I thought it would be good to have all the files separately as they are listed separately in the metadata